swagger/function-manager.yaml

```diff
@@ -224,7 +189,7 @@ paths:
           description: Internal error
           schema:
             $ref: '#/definitions/Error'
-  /{functionName}/runs:
+  /function/{functionName}/runs:
```

Comment:
I think we should remove this path.  This is equivalent to `/runs/?functionName=myfunc`

Reply:
Sure I can remove that endpoint since they're equivalent

Reply:
Well actually since we send POST requests to that path `/function/{functionName}/runs` to run a function, would it be better to just leave it then?

Reply:
Ah.  Well, I think the correct way would be to send to /runs and put the function name in the post payload.

Reply:
Sure I'll work on doing that